alf/summary/render.py

```diff
@@ -19,7 +19,10 @@
 matplotlib.use('Agg')  # 'Agg' no need for xserver!
 import matplotlib.pyplot as plt
 # Style gallery: https://tonysyu.github.io/raw_content/matplotlib-style-gallery/gallery.html
-plt.style.use('seaborn-dark')
+# The seaborn styles shipped by Matplotlib are deprecated since 3.6,
+# as they no longer correspond to the styles shipped by seaborn.
+# However, they will remain available as 'seaborn-v0_8-<style>'.
+plt.style.use('seaborn-v0_8-dark')
```

Comment:
Will matplotlib with a lower version be compatible with this option?

Reply:
As long as it is not too low. I am not sure how low is too low though, probably below 3.0.

Reply:
The problem is that if I put a `< 3.6` in our CICD docker image, it will be too restrictive. But otherwise my current PR will be blocked.

Reply:
> As long as it is not too low. I am not sure how low is too low though, probably below 3.0.

Tested it's not available in our server docker. Need to make this compatible.

Reply:
You can try and catch